internal/ttools/getenv_test.go

```diff
@@ -49,6 +52,16 @@ func TestGetEnvDefault(t *testing.T) {
 			dvalue:   "DEFAULT_VALUE",
 			expected: ENV_TEST_VARIABLE,
 		},
+		"should get variable when key is empty string": {
+			key:      "",
+			dvalue:   "DEFAULT_VALUE",
+			expected: "DEFAULT_VALUE",
+		},
+		"should get variable from empty string environment": {
+			key:      "ENV_TEST_EMPTY_VARIABLE",
+			dvalue:   "DEFAULT_VALUE",
+			expected: "DEFAULT_VALUE",
+		},
```

Comment:
I think I was not clear enough. The test is proving the exact opposite of what I was saying. I was saying if an environmental variable was set to an empty string (i.e. setting the Password to empty) it should return an empty string and not the default value. 

So, here I would expect the `expected` value to be `""`, and not `"DEFAULT_VALUE"`.

Reply:
Ah, now I got you. Thank you for your patience with me here